tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -270,6 +270,9 @@ Status ValidateTensorProperties(const string& producer_node_type,
   // Convert data type.
   TF_RETURN_IF_ERROR(TfDataTypeToTrt(dtype, trt_dtype));
 
+  if (producer_node_type == "TRTPluginOp" && validation_only) {
```

Comment:
This function also setup trt_dims for node conversion after plugin node. So we should delete this if condition here.